src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const STATE_HISTORY = [];
+  const CLONED_STATE = { ...state };
```

Comment:
The variable `CLONED_STATE` should follow camelCase naming convention and be named to reflect that it's a copy of the state. Consider renaming it to `stateCopy`.